Yoast/wordpress-seo
# Move company info / your info

## Summary

This PR can be summarized in the following changelog entry:

* Move company info / your info tab from Dashboard to Search Appearance

## Relevant technical choices:

* Split the General tab under Search Appearance into 5 different files to improve readability.

## Test instructions

This PR can be tested by following these steps:

* See that the info has moved from one page to the other, make sure everything still saves and the JSON+LD on the frontend changes along with your saved changes.

## Quality assurance


Fixes #8858
